x-pack/legacy/plugins/uptime/public/components/connected/kuerybar/kuery_bar_container.tsx

```diff
@@ -8,7 +8,7 @@ import { connect } from 'react-redux';
 import { AppState } from '../../../state';
 import { selectIndexPattern } from '../../../state/selectors';
 import { getIndexPattern } from '../../../state/actions';
-import { KueryBarComponent } from '../../functional';
+import { KueryBarComponent } from '../../functional/kuery_bar/kuery_bar';
```

Comment:
is this done to resolve circular dependency?

Reply:
Possibly? Some weirdness was happening with imports, it's possible I overlooked something. I'll try to see if I can resolve it so we can clean up this import path.